miqa/core/tests/conftest.py

```diff
@@ -15,17 +16,23 @@
 
 @pytest.fixture
 def api_client() -> APIClient:
-
     return APIClient()
 
 
 @pytest.fixture
-def authenticated_api_client(user) -> APIClient:
-    client = APIClient()
-    client.force_authenticate(user=user)
-    return client
+def authenticated_api_client(api_client, user) -> APIClient:
+    api_client.force_authenticate(user=user)
+    return api_client
+
+
+@pytest.fixture
+def staff_api_client(api_client, user_factory) -> APIClient:
```

Comment:
The reason it's staff is because that was the permission level applied on the settings endpoints, which predates this PR. (`IsAdminUser` in DRF enforces staff status, not superuser status, which I find quite surprising.)

Reply:
Agreed. I think it'll be part of a larger discussion about permissions and policies.